server/services/interventionsService.test.ts

```diff
@@ -434,6 +434,41 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
       expect(referral.furtherInformation).toBe('Some information about the service user')
     })
 
+    it('returns the updated referral when selecting the relevant sentence', async () => {
+      await provider.addInteraction({
+        state: 'There is an existing draft referral with ID of d496e4a7-7cc1-44ea-ba67-c295084f1962',
```

Comment:
Is this the same state as `There is an existing draft referral with ID of d496e4a7-7cc1-44ea-ba67-c295084f1962, and it has had a service category selected`?

Would it be fair to say this is "a newly started empty referral with ID of ..."?

Reply:
Does the newly-started state matter here? I felt that the important thing was that we simply have an existing draft referral, no matter what stage of the refer journey we're in